### DISCUSSION TITLE
How to add web fingerprint information

### DISCUSSION THREAD

**Discussion by AkunWin** [projectdiscovery/httpx#236] [Q&A] [answered]

**Comment by ehsandeep** [comment, marked as answer]
You need to use `-tech-detect` flag, make sure you are running the latest version of httpx

**Reply by AkunWin** [reply]
I would like to add some new web fingerprint info on how I should do it.

**Reply by ehsandeep** [reply]
that's not possible, httpx makes use of https://github.com/projectdiscovery/wappalyzergo library having a collection of hardcoded fingerprints.

**Reply by AkunWin** [reply]
Thank you, look forward to reading custom fingerprint support
